README.md

```diff
@@ -32,4 +32,5 @@ changed through the environment variable `RESULTCACHING_HOME`.
 | Variable | description |
 |-----------------------|----------------------------------------------------------------------------------|
 | RESULTCACHING_HOME | directory to cache results (benchmark ceilings) in, `~/.result_caching` by default |
-| RESULTCACHING_DISABLE | `'1'` to disable loading and saving of results, functions will be called directly |
+| RESULTCACHING_DISABLE | * `'1'` to disable loading and saving of results, functions will be called directly |
```

Comment:
Can you maybe add some examples? Like brainscore.assemblies and brainscore.benchmarks for benchmark diabling and candidate_models.score_models for scores and model_tools.activations for models or so?